hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -10,6 +10,17 @@ Note that 'USB mass storage boot' is different from 'USB device boot mode'. [USB
 
 If you are unable to use a particular USB device to boot your Raspberry Pi, an alternative is to use the special bootcode.bin-only boot mode as described [here](README.md). This Pi will still boot from the SD card, but `bootcode.bin` is the only file read from it.
 
+<a name="pi4"></a>
+## Raspberry Pi 4
+To enable USB mass storage boot on a Raspberry Pi 4.
```

Comment:
Personally I'd put a colon before the list of bullet points, but otherwise I'm happy with this.